homeassistant/components/philips_js/manifest.json

```diff
@@ -1,6 +1,6 @@
 {
   "domain": "philips_js",
-  "name": "Philips js",
+  "name": "Philips TV",
```

Comment:
Not really the same thing. Not all Philips tv is joinyspace capable.

Reply:
True, however, this is how we advertise it, see: <https://www.home-assistant.io/integrations/philips_js/>

So, unless you can provide a better name, I will continue on merging this for now.